cppapi/server/eventsupplier.h

```diff
@@ -247,6 +247,20 @@ public :
 	TANGO_IMP_EXP static ZmqEventSupplier *create(Util *);
 	virtual ~ZmqEventSupplier();
 
+
+    class Future
```

Comment:
Consider moving such helper classes to dedicated file so that other can use them as well in future.